Makefile.am

```diff
@@ -1270,12 +1270,26 @@ libback_ldbm_la_SOURCES = ldap/servers/slapd/back-ldbm/ancestorid.c \
 	ldap/servers/slapd/back-ldbm/db-bdb/bdb_monitor.c \
 	ldap/servers/slapd/back-ldbm/db-bdb/bdb_ldif2db.c \
 	ldap/servers/slapd/back-ldbm/db-bdb/bdb_import.c \
-	ldap/servers/slapd/back-ldbm/db-bdb/bdb_import_threads.c
+	ldap/servers/slapd/back-ldbm/db-bdb/bdb_import_threads.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_config.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_instance.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_instance_config.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_verify.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_layer.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_misc.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_perfctrs.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_upgrade.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_version.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_monitor.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_ldif2db.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.c \
+	ldap/servers/slapd/back-ldbm/db-mdb/mdb_import_threads.c
+
 
 
 libback_ldbm_la_CPPFLAGS = $(AM_CPPFLAGS) $(DSPLUGIN_CPPFLAGS) @db_inc@
 libback_ldbm_la_DEPENDENCIES = libslapd.la
-libback_ldbm_la_LIBADD = libslapd.la $(DB_LINK) $(LDAPSDK_LINK) $(NSPR_LINK)
+libback_ldbm_la_LIBADD = libslapd.la $(DB_LINK) $(LDAPSDK_LINK) $(NSPR_LINK) -llmdb
```

Comment:
Shouldn't this -llmdb be part of db_link?

Reply:
it is an option, I rather think about adding a second set of variable for lmdb and keep @db_inc@ and DB_LINK for bdb 

 using DB_LINK and replacing @db_inc@ by DB_INC should do the trick too ...

Reply:
Ok, I changed that one in last commit: replaced @db_inc@ by $(DB_INC) and moved -lldbm in DB_LINK